CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ the detailed section referring to by linking pull requests or issues.
 
 - Implemented markdown for asset descriptions
 - Implemented list view for Broker UI
+- Asset Metadata is now editable
```

Comment:
On first open the Advanced information tab does not look clickable. After clicking it once it appears differently and looks clickable

<img width="854" alt="image" src="https://github.com/sovity/edc-ui/assets/75735874/fb84736f-6ba8-4a10-9a08-3febaad09292">
<img width="844" alt="image" src="https://github.com/sovity/edc-ui/assets/75735874/b1aa0039-1cd5-4655-9a59-997a97c2b3a0">

Reply:
Known issue, to be addressed in the upcoming "asset edit dialog -> asset creation wizard" rework